desktop_version/src/editor.cpp

```diff
@@ -5981,6 +6011,21 @@ void editorinput()
             if (key.keymap[SDLK_1]) ed.drawmode=17;
             if (key.keymap[SDLK_2]) ed.drawmode=18;
             if (key.keymap[SDLK_3]) ed.drawmode=19;
+
+            if(key.keymap[SDLK_F9]) {
+				int nextspritesheet = cycleThroughCustomResources(ed.getcustomsprites(), graphics.customsprites);
+
+                ed.level[ed.levx + ed.levy*ed.maxwidth].customspritesheet = nextspritesheet;
+
+                if (nextspritesheet == 0)
+                    ed.note = "Now using default spritesheet";
+                else
+                    ed.note = "Now using sprites" + std::to_string(nextspritesheet) + ".png";
+                ed.notedelay = 45;
+                ed.updatetiles = true;
+                ed.keydelay = 6;
+				printf("Should cycle to next spritesheet\n");
```

Comment:
Debug print? Don't think we need this printed to the console...

Reply:
Also, this line, the closing bracket `}` on the next line, and the line a few lines above containing `int nextspritesheet = ...` are indented with tabs instead of spaces.

Reply:
And, the `int nexttilesheet = ...` that you added from refactoring/abstracting the customtileset cycling is also indented with tabs instead of spaces.